<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>160</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="FacialR">
       <property name="text">
        <string>FacialRight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR0">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR1">
       <property name="text">
        <string>썩소</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR2">
       <property name="text">
        <string>고양이입</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR3">
       <property name="text">
        <string>벌린 고양이입</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR4">
       <property name="text">
        <string>웃는입</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR5">
       <property name="text">
        <string>미소입</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR6">
       <property name="text">
        <string>헤에입</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR7">
       <property name="text">
        <string>으</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR8">
       <property name="text">
        <string>캬</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR9">
       <property name="text">
        <string>오</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR10">
       <property name="text">
        <string>웃는입2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR11">
       <property name="text">
        <string>메롱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnR12">
       <property name="text">
        <string>세모입</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>160</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="FacialL">
       <property name="text">
        <string>FacialLeft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL0">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL1">
       <property name="text">
        <string>화난눈</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL2">
       <property name="text">
        <string>홍조</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL3">
       <property name="text">
        <string>곤란한 검은색 빙빙이</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL4">
       <property name="text">
        <string>눈웃음</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL5">
       <property name="text">
        <string>일자눈</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL6">
       <property name="text">
        <string>에헤헿</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL7">
       <property name="text">
        <string>고양이눈</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL8">
       <property name="text">
        <string>화난홍조</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL9">
       <property name="text">
        <string>눈웃음2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL10">
       <property name="text">
        <string>반짝반짝</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL11">
       <property name="text">
        <string>썬글라스</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnL12">
       <property name="text">
        <string>윙크</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
